rollup.config.js

```diff
@@ -190,45 +194,66 @@ export function makeBaseBundleConfig(options) {
   return deepMerge(sharedBundleConfig, isAddOn ? addOnBundleConfig : standAloneBundleConfig);
 }
 
-export function makeMinificationVariants(existingConfigs) {
-  const newConfigs = [];
-
-  // ensure we've got an array of configs rather than a single config
-  existingConfigs = Array.isArray(existingConfigs) ? existingConfigs : [existingConfigs];
+/**
+ * Takes the CDN rollup config for a given package and produces three versions of it:
+ *   - non-minified, including debug logging,
+ *   - minified, including debug logging,
+ *   - minified, with debug logging stripped
+ *
+ * @param baseConfig The rollup config shared by the entire package
+ * @returns An array of versions of that config
+ */
+export function makeConfigVariants(baseConfig) {
+  const configVariants = [];
 
-  existingConfigs.forEach(existingConfig => {
-    const { plugins } = existingConfig;
+  const { plugins } = baseConfig;
+  const includeDebuggingPlugin = makeIsDebugBuildPlugin(true);
+  const noDebuggingPlugin = makeIsDebugBuildPlugin(false);
```

Comment:
For some reason, I'm having a hard time understanding the naming here. The way I understand `makeIsDebugBuildPlugin`, it either returns a plugin which includes or excludes debugging/logging. So I think we could co with something like

* `includeDebuggingPlugin` + `excludeDebuggingPlugin`
or likewise
* `debuggingPlugin` + `noDebuggingPlugin` 

What do you think?

Reply:
That's fair. You're right, the naming structure isn't exactly parallel. For whatever reason "exclude" sounds a little off to me, so I'm going to go with "include" and "strip" (since that's the verb that's normally used for pulling code out).

Reply:
Sounds good!